core/src/main/java/io/ddf/etl/ATimeSeriesHandler.java

```diff
@@ -0,0 +1,129 @@
+package io.ddf.etl;
+
+
+import io.ddf.DDF;
+import io.ddf.analytics.ABinningHandler.BinningType;
+import io.ddf.exception.DDFException;
+import io.ddf.misc.ADDFFunctionalGroupHandler;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import com.google.common.collect.Lists;
+
+public abstract class ATimeSeriesHandler extends ADDFFunctionalGroupHandler implements IHandleTimeSeries {
+
+  protected String mTimestampColumn;
```

Comment:
why do we need to keep mTimestamColumn and mTsIDColumn as instance variable, they already passed in by downsample function right?